composer.json

```diff
@@ -33,7 +33,7 @@
     "symfony/dependency-injection": "^3.4 || ^4.0",
     "symfony/framework-bundle": "^3.4 || >=4.0 < 4.1",
     "symfony/http-kernel": "^3.4 || ^4.0",
-    "symfony/messenger": "^4.2"
+    "symfony/messenger": "^4.2 < 4.3"
```

Comment:
My personal preference would be to simplify this to be `~4.2.0` which is exactly equal to what you've got here.

Reply:
Thank you, fixed!